### PR TITLE
Update apps.collection.yaml

### PR DESCRIPTION
**Pull request by chris-mercer**
+ Remove Dead projects
+ Remove multiple documented scams related to "Etcher" alias
+ Adjust so Title is on top for all

### CODE REVIEW

**Comment by gitr0n1n** [comment]
> Thanks for the PR.
> 
> I am not against this approach, but wanted to raise that a while ago, there was a discussion on how to deal with dead apps:
> 
> #862
> 
> For the sake of convenience now, removing them, as per this PR, is easiest, but another option is to flag them as dead but keep a record of them available in their own section. This will require some additional logic to deal with though.
> 
> Do maintainers have preferences on this?

I think maintaining a graveyard of apps/wallets/block explorers/exchanges and any other type of service doesn't make much sense. Productively, I think creating a section of known exit scams that have happened on Classic would be minimally helpful to the public to avoid investing in the same pseudonyms/type of scams,  e.g.: Classic Wallet/Saturn Network/Dairy Marketplace & E-Milk/Various memecoins and NFT collections.

IMO, the end users is going to this section of the website to find services that have adopted ETC. So I feel the focus of the website and these sections should be to stick to that specific use case.

We have a great archival section about the network specifically which i think is wonderful. But expanding to ALL SERVICES would become a huge task for a limit open source rag tag group of maintainers.

Just my opinion on the matter. There are already third party websites that track that type of activity, I think we keep within the scope of the goals of the website. I'd even be open to removal of the Apps section due to the frequency of scams/bad actors/deprecated projects.
https://99bitcoins.com/deadcoins/

**Comment by gitr0n1n** [comment]
What's the hold up here @ethereumclassic/website ? Let's get these scams/deprecated apps/junk out of the website. Time to clean things up for the bull cycle.

**Review by DonaldMcIntyre — APPROVED** [review]

**Pull request merged**
